### PR TITLE
Feat websocket fix

### PR DESCRIPTION
**Pull request by JacobMacFarlane**
____ Wrote Tests ____ Implemented ____ Reviewed

## Neccesary checkmarks
- [ ] All Tests are Passing in all environments
- [ X] The code will run locally
## Type of change
- [ ] New feature
- [x ] Bug Fix
- [ x] Refactor
## Description of Change:

  description:  In this PR i fixed the favorites bug so the star will mantain no matter what, went through and fixed a couple of varied ternary operators that helped in order to display the right class/value. gutted the sort by distance feature unfortunately due to lack of time
    
## Requested feedback: 
    
  I'd like feedback on a any places you think i have reused code or redundant code

## Check the correct boxes

- [ x ] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes

- [x ] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

- [ x] My code has no unused/commented out code
- [ ] My code has no binding.pry's
- [x ] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -23,67 +23,44 @@ const App = () => {
...
-    if (!favorites.find((fav) => fav.id === truck.id)) {
-      const newFavState = [...favorites, truck]
-      console.log(truck, "truck inside if statement")
-        // setFavorites(newFavState)
-    } else {
-      setVendors(vendors.filter((v) => v.favorited === true))
-        // setFavorites(newFavState)
-      }
-        console.log(truck, "truck OTHER inside if statement")
+    const newFavState = [...favorites, truck]
```
Comment (Devynnem) [inline src/Components/App/App.js:35]: Thank for figuring out this favorite status

File: src/Components/TruckDetails/TruckDetails.js [hunk 2]
```diff
@@ -70,22 +70,21 @@ const goBackHome = () => {
     //   )
     // })
 
+    let truckStatus = singularTruck.status === true ? "Open Now!" : "Closed Now!"
```
Comment (Devynnem) [inline src/Components/TruckDetails/TruckDetails.js:73]: Exciting Open and close statements!!

**Review by Devynnem — APPROVED** [review]

File: src/Components/App/App.js [hunk 3]
```diff
@@ -23,67 +23,44 @@ const App = () => {
...
     let nameSearchResults = vendors.filter((v) => v.name.toLowerCase().includes(lowerSearchValue) || v.tags.toLowerCase().includes(lowerSearchValue) || v.description.toLowerCase().includes(lowerSearchValue))
     setVendors(nameSearchResults)
-
-    //this is going to take in whatever the search value and based upon the search value it will iterate over the vendors and update the vendors state accordingly
-
-    // now if we are passing down the name of a  truck or location of truck then we just need to iteratre over vendors and filter where we see fit
   }
   
-  const searchButtons = (event, button) => {
+  const searchButtons = (event, button ) => {
```
Comment (emlindvall) [inline src/Components/App/App.js:50]: There's a random space here on line 50, just heads up!

File: src/Components/TruckCard/TruckCard.js [hunk 4]
```diff
@@ -16,30 +16,6 @@ const TruckCard = (props) => {
   let [upVote, setupVote] = useState(parseInt(props.truck.up_rating))
   const [isUpvoted, setIsUpvoted] = useState(false)
```
Comment (emlindvall) [inline src/Components/TruckCard/TruckCard.js:18]: I think taking out the distance functionality is really smart for right now, and we're all behind this! We could certainly get it working given more time, but as it stands we're losing valuable time continuing to focus on this. We can always return to it at the end of the project if it feels like we have room to play with.

File: src/Components/TruckDetails/TruckDetails.js (hunk 2 above)
Comment (emlindvall) [reply src/Components/TruckDetails/TruckDetails.js:73]: Jacob coming in HOT

**Review by emlindvall — APPROVED** [review]

**Review by Pma913 — APPROVED** [review]
I think this all looks good. Thanks for updating the favorite function. I like the clearing of the unwanted code

**Pull request merged**